tests/unit/test_router.py

```diff
@@ -156,6 +157,38 @@ def test_webhook_is_200_if_action_raises_IgnoreInvalidRequestError(
     assert response.json()["error"] == "no bug whiteboard matching action tags: devtest"
 
 
+def test_webhook_is_200_if_action_raises_Exception(
+    webhook_request_factory, mocked_bugzilla, authenticated_client
+):
+    webhook = webhook_request_factory()
+    mocked_bugzilla.get_bug.side_effect = Exception("Throwing an exception")
+
+    response = authenticated_client.post(
+        "/bugzilla_webhook",
+        data=webhook.model_dump_json(),
+    )
+    assert response
+    assert response.status_code == 200
+
+
+def test_webhook_is_500_if_queue_raises_Exception(
+    webhook_request_factory, mocked_bugzilla, authenticated_client
+):
+    # queue =
+    # is_blocked = queue.is_blocked
+    webhook = webhook_request_factory()
+    get_dl_queue().is_blocked = mock.AsyncMock(
+        side_effect=Exception("Throwing an exception")
+    )
```

Comment:
Since we're testing the route here, I think a nice "FastAPI way" to do this would be to [override the dependency](https://fastapi.tiangolo.com/advanced/testing-dependencies/#testing-dependencies-with-overrides) for this test.

Reply:
Ah that seems easy enough. Adjusting